pkg/registry/registry_antrea.csv

```diff
@@ -54,3 +54,5 @@ ElementID,Name,Abstract Data Type,Data Type Semantics,Status,Description,Units,R
 152,flowEndSecondsFromDestinationNode,unsigned32,,current,,,,,,,,56506,
 153,egressName,string,,current,,,,,,,,56506,
 154,egressIP,string,,current,,,,,,,,56506,
+155,l7FlowType,unsigned8,,current,Supported Actions(uint8 value): L7flowNotPresent(0) L7flowTypeHttp(1),,,,,,,56506,
```

Comment:
I suggested `l7FlowType` as an example, but from a consistency standpoint with standard IPFIX IEs, I feel like `l7ProtocolName` is a better name

I also feel like using a short string (`"http"`, and maybe later `"grpc"`, `"ssh"`, etc) makes more sense than using an integer value, unless these integer values are standardized somewhere (I doubt it).

Reply:
I see, I got confused as it was mentioned enum, allow me to change it.